docs: reorganise (#113)
algolia/algoliasearch-netlify: docs/FAQ.md

# FAQ

> Here is general purpose FAQ. If you are looking for technical answer, please go to our [Technical documentation](Extraction.md).

Reviewer: ```suggestion
Here is general purpose FAQ. If you are looking for technical answers, please go to our [Technical documentation](Extraction.md).
```

> Here is general purpose FAQ. If you are looking for technical answers, please go to our [Technical documentation](Extraction.md).

Reviewer: ```suggestion
This is a general purpose FAQ. If you are looking for technical answers, go to our [Technical documentation](Extraction.md).
```

## Can I build my own UI?<!-- omit in toc -->

Of course!

We recommend using InstantSearch, our open-source, production-ready UI libraries that let you quickly build a search interface in your front-end application. InstantSearch is available for [vanilla JS](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/), [React](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Vue](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/vue/) and [Angular](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/angular/).

## Can I receive a notification when my crawl is done?<!-- omit in toc -->

You can configure notifications in the [account settings](https://crawler.algolia.com/admin/user/settings/) of your Crawler Admin Console.

## My website is not crawled<!-- omit in toc -->

The monitoring tab in the Crawler UI will provide information to help you improve the crawler behavior.
By default we respect most of the standard web rules: robots.txt, password protection, canonical, redirection, etc...

You can help the crawler by:

> - Allowing our user-agent in [your robots.txt](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-ip-address-can-i-use-for-ip-whitelisting)

Reviewer: ```suggestion
- Allowing our user-agent in [your robots.txt](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-is-the-user-agent-of-the-crawler-useful-for-whitelisting)
```

> - Allowing our user-agent in [your robots.txt]([<!-- omit in toc -->](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-ip-address-can-i-use-for-ip-whitelisting))

Reviewer: ```suggestion
- Allowing our user-agent in [your robots.txt](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#what-is-the-user-agent-of-the-crawler-useful-for-whitelisting))
```

- Specifying relative URLs in your links and canonical instead of absolute URLs that could conflict with the domain you may use.

## I have a custom domain<!-- omit in toc -->

The plugin currently only support custom domains that are configured in Netlify (**Settings > Domain management > Custom domains**).

## Do I need to display the Algolia logo?

Yes, but only if you are on a free plan.
Please see our official answer [here](https://www.algolia.com/doc/faq/accounts-billing/do-i-need-to-display-the-algolia-logo-when-i-am-on-the-free-plan/).

## I am an Algolia paying customer, can I use my own Application?

> Yes, but it currently require a manual intervention.

Reviewer: ```suggestion
Yes, but it currently requires a manual intervention.
```
